<ui version="4.0" >
 <class>KopeteStatusEditWidget</class>
 <widget class="QWidget" name="KopeteStatusEditWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>228</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Title:</string>
     </property>
     <property name="buddy" >
      <cstring>statusTitle</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KLineEdit" name="statusTitle" />
   </item>
   <item>
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Message:</string>
     </property>
     <property name="buddy" >
      <cstring>statusMessage</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="statusMessage" />
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="setButton" >
       <property name="text" >
        <string>Set</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="clearButton" >
       <property name="text" >
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header location="global" >kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global" >klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
